include/entity.h

```diff
@@ -737,7 +737,7 @@ typedef struct {
 
 typedef struct {
     /* 0x7C */ u8 pad0[0x4];
-    /* 0x80 */ s32 unk80;
+    /* 0x80 */ u8* anim;
```

Comment:
Seems like an alignment bug is possible here but didn't see a clear example in the entity code